### PR TITLE
Test setup

### LINKED ISSUE #42: Update userLogin query to include style attribute
- [ ] userLogin query will need to have style attribute added to that query 

```
style:  {
    name: 'style',
    type: GraphQLString
}
```

### PR DESCRIPTION
**Pull request by Stearnzy**
### Description
Add Jest and Supertest
Add test DB block
Add userStyleUpdate mutation
Add styleUpdate resolver
Add query tests, specifically for user with null style
Setup for other testing

### Closes issue(s)
Closes #42

### Testing
Some tests made, others will be elaborated on

### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### Checklist
- [x] I have written tests for this code (happy & sad)
- [ ] I have updated the Readme
- [ ] I ran full test suite - all tests passing

### CODE REVIEW

File: code/api/src/modules/user/mutations.js [hunk 1]
```diff
@@ -37,4 +37,20 @@ export const userRemove = {
     }
   },
   resolve: remove
-}
\ No newline at end of file
+}
+
+// Update Style
+export const userStyleUpdate = {
```
Comment (Shawntru) [inline code/api/src/modules/user/mutations.js:43]: Cool, I see this is what we'll be calling to modify the existing user style.

**Review by Shawntru — APPROVED** [review]
Clearly written and testing looking good!
